src/components/Tag/TagAddNewTagModal.jsx

```diff
@@ -63,6 +64,7 @@ const TagAddNewTagModal = ({ onClick, onClose, withLabel }) => {
             busy={isBusy}
             disabled={label.length === 0}
             onClick={handleClick}
+            data-testid="TagAddNewTagModal-Button-submit"
```

Comment:
is it mandatory to use data-testid only for test purpose?

isn't it possible to use getByRole('button') or getAllByRole('button')[0] ?

Reply:
> nit: is it mandatory to use data-testid only for test purpose?

Pas obligatoire, mais c'est beaucoup plus clair ainsi 👍